upload/catalog/controller/extension/payment/instamojo.php

```diff
@@ -36,12 +36,39 @@ public function start() {
 	$order_info = $this->model_checkout_order->getOrder($this->session->data['order_id']);
 	
 	$this->logger->write("Step 2: Creating new Order with ".$this->session->data['order_id']);
+
+	    if ($order_info['currency_code'] != "INR"){
+
+		$this->load->model('localisation/currency');
+        	$currencies = $this->model_localisation_currency->getCurrencies();
+
+        foreach ($currencies as $currency) { 
+			if ($currency['code'] == $order_info['currency_code']) { 
+            		$order_info['total'] == (float) ($order_info['total'] / $currency['value']) ; 
+            		$order_info['total'] = round($order_info['total'],2);
```

Comment:
can we add a break statement here?
also, what happens if the currency != INR and is not found in getCurrencies too?

Reply:
also, what happens if the currency != INR and is not found in getCurrencies too?

Reply:
Will do.
Then, it will take the value of the default currency. When INR is the default value, the value will not change.